tests/test_core.py

```diff
@@ -36,7 +36,7 @@ def setUp(self):
 
     def test_str(self):
         emp = Emperor(self.ord_res, self.data, self.headers)
-        self.assertEqual(HTML_STRING, str(emp))
+        self.assertItemsEqual(HTML_STRING.split('/'), str(emp).split('/'))
```

Comment:
Can you update this test case in the same way that you updated the test case in format.py?